admin_interface/apps.py

```diff
@@ -11,3 +11,6 @@ def ready(self):
         from admin_interface import settings
 
         settings.check_installed_apps()
+
+        # must check if LOCAL_FILE_DIR is set in settings
+        settings.check_settings("LOCAL_FILE_DIR")
```

Comment:
This setting should be optional, it makes no sense to ask users to set a setting that they probably don't need.